packages/EasySchedule/src/Bridge/Doctrine/CommandExecutedEventListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasySchedule\Bridge\Doctrine;
+
+use Doctrine\ORM\EntityManagerInterface;
+use EonX\EasySchedule\Event\CommandExecutedEvent;
+
+final class CommandExecutedEventListener
```

Comment:
Let's rename to `DoctrineClearEntityManagerOnCommandSubscriber`

Reply:
We usually name listeners according to the event name. Only subscribers should have some purpose-based names. So let's keep the current name.

Reply:
Maybe then use a subscriber?

Reply:
Ok. Got it.